### ISSUE TITLE
There is no link to the complete lesson at the top of the repo yet

### ISSUE THREAD

**Issue by RMHogervorst** [datacarpentry/textmining-socialsci#4]

**Comment by benmarwick** [comment]
Right, currently the lessons can be browsed via the Rmd files in this repo. Github renders them quite nicely, so they're easy to read.

**Comment by RMHogervorst** [comment]
What I meant was: in the software carpentry lessons every github repo has a link to the working website that is not yet available for this repo. 
In fact I can not find workng link

**Comment by RMHogervorst** [comment]
For instance: http://www.datacarpentry.org/spreadsheet-ecology-lesson/
https://github.com/datacarpentry/spreadsheet-ecology-lesson/
